### PR TITLE
Added New Component In HireWithUs

### LINKED ISSUE #201: Create Hire-with-us component-2
Here is the component design
image
Here is the full prototype: #199

### PR DESCRIPTION
**Pull request by atharmohammad**
Fixes #201

### CODE REVIEW

**Comment by atharmohammad** [comment]
@Abhishek-kumar09  Can you please tell me Why checks are Failing,Thanks

File: src/views/pages/HireWithUsView/Developer/DevBox/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    }
+    default:img=Thumbs_Up
+  }
+
+  return (
+    <div>
+      <div className={classes.DevBox}>
+        <div className={classes.Content}>
+          <div>
+            <img src={img} className={classes.Gold_Ribbon}/>
```
Comment (Abhishek-kumar09) [inline src/views/pages/HireWithUsView/Developer/DevBox/index.js:31]: `img` should have a alt prop.

**Comment by atharmohammad** [comment]
@Abhishek-kumar09  Thanks man, Look at changes

File: public/index.html [hunk 2]
```diff
@@ -4,15 +4,17 @@
     <meta charset="utf-8" />
     <meta name="color-scheme" content="dark light">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital@1&display=swap" rel="stylesheet">
```
Comment (Abhishek-kumar09) [inline public/index.html:8]: Why these changes?

File: src/views/pages/HireWithUsView/Developer/Developer.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            <div className={classes.Box}>
+              <Devbox Img='Gold_Ribbon'/>
+              <Devbox Img='Book_Icon'/>
+              <Devbox Img='Thumbs_Up'/>
+          </div>
+
+          <div className={classes.Bottom}>
+            <Devbox/>
+            <div className={classes.BottomBox}>
+              Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset ke Aldus PageMaker including versions of Lorem Ipsum elease of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop.
```
Comment (Abhishek-kumar09) [inline src/views/pages/HireWithUsView/Developer/Developer.js:27]: Actually you can use Typography, which is already defined in `material-ui`. https://github.com/codeforcauseorg/codeforcause.org/blob/development/src/theme/typography.js

You have a look on the documentation here: https://material-ui.com/api/typography/

File: public/index.html (hunk 2 above)
Comment (atharmohammad) [reply public/index.html]: I added a google font link ,removed it in this commit

File: src/views/pages/HireWithUsView/Developer/Developer.js (hunk 3 above)
Comment (atharmohammad) [reply src/views/pages/HireWithUsView/Developer/Developer.js]: Ok , added it in the component , review it

File: src/views/pages/HireWithUsView/Developer/DevBox/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+  return (
+    <div>
+      <div className={classes.DevBox}>
+        <div className={classes.Content}>
+          <div>
+            <img src={img} className={classes.Icon} alt=""/>
+          </div>
+          <strong><p>1000+</p></strong>
+          <strong><p>Problems Solved</p></strong>
```
Comment (Abhishek-kumar09) [inline src/views/pages/HireWithUsView/Developer/DevBox/index.js:34]: Please use Typogaphy here.

File: src/views/pages/HireWithUsView/Developer/Developer.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  render(){
+
+    return (
+
+      <div>
+        <div className={classes.Developer}>
+
+        <div>
+         <p><span className={classes.heading1}>The Making Of </span>
+           <span className={classes.heading2}>The Fine Developer</span></p>
```
Comment (Abhishek-kumar09) [inline src/views/pages/HireWithUsView/Developer/Developer.js:16]: PLease use typography here.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
We use material-ui for our styling and use typography where we have to show some text so that the code is more consistent and less written. Changes are good but minor changes for now.

File: src/views/pages/HireWithUsView/Developer/Developer.js (hunk 5 above)
Comment (atharmohammad) [reply src/views/pages/HireWithUsView/Developer/Developer.js]: Added Tell me if more Changes required

File: src/views/pages/HireWithUsView/Developer/DevBox/index.js (hunk 4 above)
Comment (atharmohammad) [reply src/views/pages/HireWithUsView/Developer/DevBox/index.js]: added

**Comment by Abhishek-kumar09** [comment]
I can't see the vertical transparent glass in my web : 
image
But is seen in your screenshot: https://user-images.githubusercontent.com/56029409/107119614-da3d9800-68ae-11eb-8a76-51d3e6105d10.PNG

**Comment by Abhishek-kumar09** [comment]
https://deploy-preview-212--eloquent-turing-542749.netlify.app/hirewithus
It is still not visible, If you have added and it is not visible then the problem may be with `zIndex` probably.

**Comment by atharmohammad** [comment]
> I can't see the vertical transparent glass in the extreme right in my web :
Even though it is visible in my web browser , may be its because of zoom difference I commit some changes , try it now ...try 100% zoom of browser window

**Comment by atharmohammad** [comment]
@Abhishek-kumar09  SideBar css is used for that ...I have already given it larger z-index

**Comment by atharmohammad** [comment]
> https://deploy-preview-212--eloquent-turing-542749.netlify.app/hirewithus
> It is still not visible, If you have added and it is not visible then the problem may be with `zIndex` probably.

I am looking into it ... Do my changes appears here as i commit so can see if it appears or not

**Comment by atharmohammad** [comment]
@Abhishek-kumar09  Ok,So opactiy was the problem Which was supported in my browser by 80% so I have Now changed it to 0.8 Now You can see the side bar,Thanks

**Comment by Abhishek-kumar09** [comment]
Cool, these changes are good forward leap. Could you use material-ui styles it's same as the current css all the properties work same there too, just we have everything in one file so that it is more organized. There are plenty of files that uses material-ui styles you can definately look into that how it is done. Meanwhile you will get a chance to learn about it too.

**Comment by atharmohammad** [comment]
@Abhishek-kumar09 yes, I saw that  in the code , will convert css as per the  application structure 👍

**Comment by atharmohammad** [comment]
@Abhishek-kumar09  Done, Review Changes,Thanks 👍

**Comment by Abhishek-kumar09** [comment]
Looks good, thanks for the PR @atharmohammad.

**Pull request merged**